src/main/kotlin/no/nav/integrasjon/api/nais/client/NaisClient.kt

```diff
@@ -47,14 +47,14 @@ fun Routing.getIsReady(adminClient: AdminClient, config: FasitProperties) =
                 )
                     Pair(HttpStatusCode.OK, "is ready")
                 else
-                    Pair(HttpStatusCode.ExceptionFailed, "is not ready")
+                    Pair(HttpStatusCode.ExpectationFailed, "is not ready")
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/417

> The HTTP 417 Expectation Failed client error response code indicates that the expectation given in the request's Expect header could not be met.

As far as I can see, we do not use or require any `Expect` headers.

The 4xx series of HTTP error codes indicates a client error - I think it'd be more semantically correct to send back a 500 Internal Server Error instead if a request results in an exception.

Same comment for every other usage of this HTTP status code.

Reply:
Also, if the request failed due to an error in the request (e.g. missing or wrong parameters), a HTTP 400 should be returned.

Reply:
Concur :-)